source/Missions/Services/Taverns/CoopTavernsController.cs

```diff
@@ -23,32 +24,32 @@ internal class CoopTavernsController : MissionBehavior
         private static readonly ILogger Logger = LogManager.GetLogger<CoopArenaController>();
         public override MissionBehaviorType BehaviorType => MissionBehaviorType.Other;
 
-        private readonly IMessageBroker _messageBroker;
+        private readonly INetworkMessageBroker _messageBroker;
```

Comment:
Convert this back to `IMessageBroker`

Reply:
Cannot do, needed for instantiating BoardGameManager which will be done via DI eventually